olxcleaner/loader/xml_exceptions.py

```diff
@@ -69,7 +69,7 @@ class InvalidPointer(CourseError):
 
     def __init__(self, filename, **kwargs):
         super().__init__(filename)
-        self._description = f"The {kwargs['edxobj']} tag looks like it is an invalid pointer tag"
+        self._description = f"The {kwargs['edxobj']} tag should not be empty and point to another XML file at the same time."
```

Comment:
@awaisdar001 Please review the updated message for `InvalidPointer`

Reply:
@jolyonb thoughts ^

Reply:
The error message here should probably be something like "The ... tag has already been pointed to and is not allowed to point to another file." The other error message should be something like "The ... tag has been pointed to and is unexpectedly empty."

Reply:
I like `"The ... tag has already been pointed to and is not allowed to point to another file."` 👍